structs.go

```diff
@@ -17,6 +17,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/bwmarrin/discordgo/internal/ratelimit"
```

Comment:
Just move ratelimit into the dgo directory, not sure we need to have an internal package.